phpBB/styles/prosilver/theme/common.css

```diff
@@ -1266,7 +1300,8 @@ ul.linklist:after,
 	content: "";
 }
 
-.dropdown-extended .footer {
+.dropdown-extended .footer,
+.smiley-list .footer {
```

Comment:
yes i see that just noticed that the dropdown-extended and notifications classes are horribly faulty and am working on a pr to fix them but for know fix the color using the smiley classes